ui/app/AppLayouts/ActivityCenter/ActivityCenterLayout.qml

```diff
@@ -358,7 +358,10 @@ StatusSectionLayout {
             notification: parent.notification
             community: notification ? root.store.getCommunityDetailsAsJson(notification.communityId) : null
 
-            onSetActiveCommunityRequested: (communityId) => { root.store.setActiveCommunity(communityId) }
+            onSetActiveCommunityRequested: (notificationId, communityId) => {
+                root.store.setActiveCommunity(communityId)
+                root.activityCenterStore.markActivityCenterNotificationRead(notificationId)
```

Comment:
Wondering if it would make sense to directly use the `notification.id` instead? Indeed, also same question for the `communityId`.. I'll create a task to review it in general.. bc we are passing params from top to bottom and the opsite direction that maybe are not necessary..